lighthouse-core/test/lib/network-request-test.js

```diff
@@ -0,0 +1,243 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const NetworkRequest = require('../../lib/network-request');
+const assert = require('assert');
+
+/* eslint-env jest */
+describe('network request', function() {
+  function getRequest() {
+    const req = new NetworkRequest();
+    req.transferSize = 100;
+    req.responseHeaders = [{name: NetworkRequest.HEADER_FETCHED_SIZE, value: 10}];
+    return req;
+  }
+
+  describe('update transfer size for lightrider', function() {
+    it('does nothing if not Lightrider', function() {
+      const req = getRequest();
+      global.isLightrider = false;
```

Comment:
need to remove `global.isLightrider` (which will bleed between tests when running `jest` with `--runInBand`, which we do in CI). Could just do `global.isLightrider === undefined` in an `afterAll`.

Reply:
bump on this

Reply:
Yup good call, totally missed this on first pass.